src/components/TestToolRow.js

```diff
@@ -14,7 +14,7 @@ const propTypes = {
 
 const TestToolRow = props => (
     <View style={[styles.flexRow, styles.mb6, styles.justifyContentBetween, styles.alignItemsCenter]}>
-        <View style={styles.flex3}>
+        <View style={styles.flex2}>
```

Comment:
Why this?

Reply:
There are multiple issues there but I think you are referring to Preferences Page.

Reply:
But we updated the Button view style to fix that one, not sure how this fixed it. Can you briefly explain this thanks!

Reply:
>Can you briefly explain this

Ok, this is what you get when using `styles.flex3` with a large font size.

![Simulator Screen Shot - iPhone 13 - 2022-11-21 at 20 40 04](https://user-images.githubusercontent.com/51829206/203134587-2d930f53-ba09-4cfa-baed-5dbd6afd9bef.png)


And this is what you get when using `styles.flex2` with a large font size.


![Simulator Screen Shot - iPhone 13 - 2022-11-21 at 20 40 13](https://user-images.githubusercontent.com/51829206/203134678-4d129813-f36c-4f2c-994a-9632baf5dac4.png)

As you can see, there is a difference between both, by using `styles.flex2` you can see the `Invalidate` word completely.